src/utility/Auth.js

```diff
@@ -0,0 +1,55 @@
+import React, { createContext, useContext, useState, useEffect } from 'react'
+import firebase from 'firebase/app'
+import 'firebase/auth'
+
+const AuthContext = createContext()
```

Comment:
why is AuthContext empty here?

Reply:
There is no data for the auth context yet. It is initialized in the AuthProvider component.